Assets/Scripts/RandomFlash.cs

```diff
@@ -3,10 +3,11 @@
 using UnityEngine;
 
 public class RandomFlash : MonoBehaviour {
-
+	Vector3 lightPosition;
 	// Use this for initialization
 	void Start () {
-		
+		lightPosition = new Vector3 (0, 0, 0);
+		InvokeRepeating ("ChangePosition", 0, 2);
```

Comment:
Please add a comment here so that in the future we know InvokeRepeating() is only temporary, and needs to change to something else.

Reply:
Suggestion accepted.